guides/policy/azure-authenticator/index.html

```diff
@@ -0,0 +1,140 @@
+<h1>Azure Authenticator Policy Generator</h1>
+
+Generate Host Policy needed to use Azure Authenticator using Azure token
+
+<br />
+<br />
+
+<table class="table table-striped border" id="tblInput">
+    <thead>
+        <tr>
+            <th scope="col-1">#</th>
+            <th scope="col-2">Field</th>
+            <th scope="col-2">Value</th>
+            <th scope="col-7">Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <th scope="row">1</th>
+            <td>Application ID</td>
+            <td><input type="text" id="txtAppId"></td>
+            <td>The ID you want to use in the policy to identify this application.
+            Used as host ID.</td>
+        </tr>
+    </tbody>
+</table>
+
+<br />
+
+Azure access token
+<textarea class="container text-monospace small border" style="height: 291px" id="encodedToken"></textarea>
+
+<br />
+
+</p>
+<div class="show" id="collapseInfo">
+    <div class="card card-body">
+
+        <pre class="container border" id="theTemplate" hidden="hidden">
+      - !host
+        id: {{app ID}}
+        annotations:
+          authn-azure/subscription-id: {{subscription ID}}
+          authn-azure/resource-group: {{resource group}}
+          authn-azure/{{user or system}}-assigned-identity: {{identity}}
+</pre>
+
+        <h3>Policy</h3>
+        <pre class="container border" id="output">
+</pre>
+
+    </div>
+</div>
+
+<script>
+
+    $(document).ready(function () {
+
+        $('#encodedToken').on('input', function () {
+            updateCmd();
+        });
+
+        $('#tblInput input').on('input', function () {
+            updateCmd();
+        });
+    });
+
+    function updateCmd() {
+        let output = "";
+        let appId = $("#txtAppId").val();
+        let encodedToken = $("#encodedToken").val();
+        if (!appId){
+            output = "Application ID is missing";
+        }
+        else if (!encodedToken){
+            output = "Azure token is missing";
+        }
+        else {
+            output = getPolicyFromToken(encodedToken, appId);
+        }
+        $("#output").html(output);
+    }
+
+    function getPolicyFromToken(encodedToken, appId) {
+        try {
+            let policy = $("#theTemplate").html();
```

Comment:
`.html` is a selector where it only takes the first match. So if you have many `theTemplate` elements, it will only take the first HTML element that matches the query. Is that what you want? If not consider using `querySelectorAll`.

Reply:
Used js template instead.